contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcMeasureConstants.java

```diff
@@ -248,7 +394,9 @@
    * {@link Measure} for server RPC request message counts.
    *
    * @since 0.8
+   * @deprecated in favor of {@link #GRPC_SERVER_SENT_MESSAGES_PER_RPC}.
```

Comment:
GRPC_SERVER_SENT_MESSAGES_PER_RPC and GRPC_SERVER_RECEIVED_MESSAGES_PER_RPC are also swapped.